cmd/tink-cli/cmd/get/get.go

```diff
@@ -0,0 +1,100 @@
+package get
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/jedib0t/go-pretty/table"
+	"github.com/spf13/cobra"
+)
+
+type CmdOpt struct {
+	// Headers is the list of headers you want to print as part of the list
+	Headers []string
+	// RetrieveData reaches out to Tinkerbell and it gets the required data
+	RetrieveData func(context.Context) ([]interface{}, error)
+	// PopulateTable populates a table with the data retrieved with the RetrieveData function.
+	PopulateTable func([]interface{}, table.Writer) error
+
+	// Output specifies the output you want the list of resources printed
+	// out. By default it is table but it can be JSON ar CSV.
+	Output string
```

Comment:
I used `-o/--output` because kubectl uses it. Docker uses `--format` without a short version.

Now, `create` uses or will require a way to pass a file or a path as we are discussing here #270. if you want `--format/-f` we won't be able to use `--file/-f` in creating/update. Any opinion? My suggestions is:

* `--format` without a short version
* `-f/--file` to pass a filepath during create/updater

Reply:
Yep I like --format & -f/--file that sounds ok to me. We can also do -F for format too.

Reply:
I'm also swayed by `kubectl` usage. `-o/--output` or `--format` works for me, I also like `-f/--file`.  Config files are the only other use-case that comes to mind and `--config-something` can work for that.

I can never get `-O` or `-o` straight between `curl` and `wget`, while redirection works in most cases, sometimes it is handy to have an output file argument in cases where you don't control the environment.  I think we can wait for that request to come along before introducing the feature.

The `packet-cli` went with `--json` and `--yaml`, which would have been fine but `-y` and `-j` were added as short names (🤦  mea culpa) which limits our ability to use those later, my advise is to not introduce a short name until you are sure about it.

Reply:
I'm ok with -o but its not my preference, if we do go with it I think tacking "format" to it in all its references makes sense (cli -h output, OutputFormat struct field...) it just seems more descriptive to use format over output imo :D.

I don't think we'd want to replace -f/--file with --config... as its not meant for config and is actually scope pretty well by the <resource> parent anyway. -i/--input is probably a better option name.